build.sbt

```diff
@@ -142,7 +142,7 @@ lazy val api = (project in file("."))
         oldStrategy(x)
     },
     name in Universal := s"${Constant.organisation}-${Constant.appName}",
-    packageName in Universal := s"${Constant.organisation}-${Constant.appName}",
+    packageName in Universal := s"${Constant.organisation}-${Constant.appName}-${Versions.appVersion}",
```

Comment:
Use sbr-api assembly approach and the SettingKey version, such that use s"${organizationName.value}-${moduleName.value}-%<PACKAGE_TYPE>%-${version.value}.%<EXTENSION>%".